open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java

```diff
@@ -0,0 +1,194 @@
+package io.opensphere.core.control.keybinding;
+
+import io.opensphere.core.util.FontIconEnum;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+
+/**
+ * Contains all the methods used for generating the panels and custom objects
+ * within the "Key Map" sub tab named "Controls"
+ */
+public class ControlTab
+{
+    /**
+     * The JavaFX gridpane containing the shortcut key keys and keys in a column
+     * arranged fashion. Since this pane is a child of another pane it must be
+     * accessed via the get method when adding items to it outside of this
+     * class.
+     */
+    private GridPane myDisplayArea;
+
+    @SuppressWarnings("unused")
+    public Button customIco(FontIconEnum text)
+    {
+        Button theButton = new Button();
+
+        final Label label = new Label(text.getFontCode());
+        Color color = Color.WHITE;
+        label.setTextFill(color);
+        label.setOpacity(75);
+
+        final String fontName = "icons-" + text.getFont().getFontName().replaceAll("\\s", "-");
+        label.getStyleClass().addAll(fontName);
+        label.setStyle("-fx-font-size: 16 px;");
+
+        theButton.setGraphic(label);
+        theButton.setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
+        theButton.setDisable(true);
+        theButton.setMaxSize(22, 22);
+        return theButton;
+    }
+
+    @SuppressWarnings("unused")
```

Comment:
Looks like it is actually unused, so just remove the method.

Reply:
it is used; but removed warning